core/src/apps/cosmos/transaction.py

```
@@ -213,34 +220,17 @@
     def deserialize(raw_message: bytes) -> "Transaction":
-        j = json.loads(raw_message.decode("utf-8"))
-        if (
-            "account_number" not in j
-            or "chain_id" not in j
-            or "fee" not in j
-            or "memo" not in j
-            or "msgs" not in j
-            or "sequence" not in j
+        try:
+            j = json.loads(raw_message.decode())
+        except Exception:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)